src/app/clusters/fan-control-server/fan-control-server.cpp

```diff
@@ -198,7 +198,7 @@ MatterFanControlClusterServerPreAttributeChangedCallback(const ConcreteAttribute
                 }
                 else
                 {
-                    res = Status::WriteIgnored;
+                    res = Status::InvalidInState;
```

Comment:
We apparently have no test coverage, since nothing started failing.

Please add a basic YAML test that exercises this.

Reply:
I've modified the "TestFanControl.yaml" file to include coverage for this particular case.